1.Core/Helpers/ProjectContextMenuHelper.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Linq;
+
+namespace Todoly.Core.Helpers;
+
+public class ProjectContextMenuHelper
```

Comment:
This class could be placed on the Views project, because the issue with it being placed here is that Core should be decoupled of any AUT, in our case, decoupled of todo.ly

Reply:
ok, done